PyInstaller/hooks/hook-shapely.py

```diff
@@ -21,7 +21,12 @@
 if compat.is_win:
     binaries = []
     lib_dir = os.path.join(pkg_dir, 'DLLs')
-    binaries += [(os.path.join(lib_dir, f), '') for f in os.listdir(lib_dir)]
+    if os.path.isdir(lib_dir):
+        binaries += [(os.path.join(lib_dir, f), '') for f in os.listdir(lib_dir)]
+    else:
+        # conda-forge package
+        lib_dir = os.path.join(pkg_dir, '..', '..', '..', '..', 'Library', 'bin')
+        binaries += [(os.path.join(lib_dir, f), '') for f in ['geos_c.dll', 'geos.dll']]
```

Comment:
- Why does this only include specific dlls, while the code above includes all in lib_dir?
- Sure this is correct? While officially undocumented, the code says "If no top-level target directory was passed, default this to the basename of the top-level source directory.". Thus for conda the dlss will end in _MEIPATH/bin. (I have not tested this, though).

Reply:
For why this does just 2 dlls instead of all the files in the path: This `lib_dir` is where all libraries are stored. The directory in the other if statement is *just* for shapely (see `get_package_paths` use at top).

Reply:
As for the top-level target directory stuff, you mean you think this is not correct to have `''` as the target? This is how every other hook does it. Everything gets dumped in the root directory.